enrolments/tests/test_serializers.py

```diff
@@ -0,0 +1,68 @@
+from django.test import TestCase
+from rest_framework.test import APIRequestFactory
+
+from accounts.models import User
+from registration.models import Family
+from registration.models import Student
```

Comment:
nit: can we merge these two lines? something like `from registration.models import Family, Student`

Reply:
Yes! Good catch